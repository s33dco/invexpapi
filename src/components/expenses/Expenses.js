import React, { Fragment } from 'react';

const Expenses = () => {
	return (
		<Fragment>
			<h1>Expenses</h1>
		</Fragment>
	);
};

export default Expenses;
