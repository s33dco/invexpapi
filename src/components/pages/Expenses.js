import React, { Fragment } from 'react';

const Expenses = () => {
	return (
		<Fragment>
			<p>Expenses</p>
		</Fragment>
	);
};

export default Expenses;
