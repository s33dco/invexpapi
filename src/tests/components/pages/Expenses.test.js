import React from 'react';
import { shallow } from 'enzyme';
import Expenses from '../../../components/pages/Expenses';

let wrapper;

beforeEach(() => {
	wrapper = shallow(<Expenses />);
});

describe('Expenses component', () => {
	it('should render Expenses component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
