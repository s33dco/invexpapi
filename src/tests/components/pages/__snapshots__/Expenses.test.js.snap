// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expenses component should render Expenses component correctly 1`] = `ShallowWrapper {}`;
